### PR TITLE
Add missing Wikidata data types

### PR DESCRIPTION
**Pull request by Mystou**

### CODE REVIEW

File: wikidataintegrator/wdi_core.py [hunk 1]
```diff
@@ -2798,6 +2799,336 @@ def from_json(cls, jsn):
...
+        super(WDTabularData, self).__init__(value=value, snak_type=snak_type, data_type=self.DTYPE,
+                                         is_reference=is_reference, is_qualifier=is_qualifier, references=references,
+                                         qualifiers=qualifiers, rank=rank, prop_nr=prop_nr,
+                                         check_qualifier_equality=check_qualifier_equality)
+
+        self.set_value(value=value)
+
+    def set_value(self, value):
+        assert isinstance(value, str) or value is None, "Expected str, found {} ({})".format(type(value), value)
+        self.value = value
```
Comment (Mystou) [inline wikidataintegrator/wdi_core.py:2893]: Add format checking here.
Value must start with <code>Data:</code> and end with <code>.tab</code>. In addition title should not contain characters like colon, hash or pipe.

**Comment by Mystou** [comment]
Add geo-shape format check.
Value must start with <code>Data:</code> and end with <code>.map</code>. In addition title should not contain characters like colon, hash or pipe.

**Comment by Mystou** [comment]
EntitySchema data type is missing but I lack of informations for this one so it will be in a future PR.

**Comment by andrawaag** [comment]
My understanding of EntityScehma is that it is not yet a proper datatype in Wikidata yet, but more an extension to Mediawiki that works in very close collaboration with Wikibase. I also understand that it is not yet possible to write EntitySchema's to Wikidata through its API. Any directions to the contrary would be appreciated.

**Comment by Mystou** [comment]
> My understanding of EntityScehma is that it is not yet a proper datatype in Wikidata yet, but more an extension to Mediawiki that works in very close collaboration with Wikibase. I also understand that it is not yet possible to write EntitySchema's to Wikidata through its API. Any directions to the contrary would be appreciated.

Thank you for the merge.

**Pull request merged**